src/CMakeLists.txt

```diff
@@ -128,7 +128,7 @@ if(UNIX)
 
   add_definitions(-DUNIX)
 
-  find_library(LIB_PTHREAD pthread)
+  find_package (Threads)
```

Comment:
Could you remove the space before the opening bracket, please?

Reply:
sure